src/Symfony/Component/Workflow/Tests/SupportStrategy/InstanceOfSupportStrategyTest.php

```diff
@@ -31,7 +32,7 @@ public function testSupportsIfNotClassInstance()
         $this->assertFalse($strategy->supports($this->createWorkflow(), new Subject1()));
     }
 
-    private function createWorkflow()
+    private function createWorkflow(): MockObject
```

Comment:
this should actually be `MockObject&Workflow`, or just `Workflow` as usages of this method care more about getting a Workflow than about getting a MockObject.